### DISCUSSION TITLE
_hf but on connected record value

### DISCUSSION THREAD

**Discussion by Deanobeano2** [cortexrd/Knack-Toolkit-Library#114] [Q&A]
Hi

I'm wondering if the use of the _hf keyword can be expanded so that a field in the detail/child part of the form can be hidden based on the value of the parent/connected record, as it doesn't appear that this can be achieved using display rules.

In our case some Orders might be for 1 Location only and if the Order has data enter in the Location field then we'd like to hide the Location field in the form for the Line Items as we populate the Line Item Location using Record Rules  i.e. Hide the Line Item Location field in the form if the Order Location field is not blank.

Thanks
Dean

**Comment by amandajane-mo** [comment]
Hi Jon,

Yes we're having trouble too, one of our apps is working fine the other same issue as you. We're having a go at fixing it while we await Norms fix.

Amanda
